feature_store_api_client/api/default/get.py

```diff
@@ -1,86 +1,93 @@
 from typing import Any, Dict, Optional, Union, cast
 
+from http import HTTPStatus
+
 import httpx
 
-from ...client import Client
+from ... import errors
+from ...client import AuthenticatedClient, Client
 from ...models.feature_store import FeatureStore
 from ...types import Response
 
 
-def _get_kwargs(
-    *,
-    client: Client,
-) -> Dict[str, Any]:
-    url = "{}/".format(client.base_url)
+def _get_kwargs() -> Dict[str, Any]:
```

Comment:
Will not returning things like the headers, cookies, etc. like before cause any issues?

Reply:
Nice catch! I regenerated the client again and seems this is brought back.